sycl/doc/extensions/proposed/sycl_ext_oneapi_group_load_store.asciidoc

```diff
@@ -391,10 +392,42 @@ For example, we can assert that `input` is a contiguous iterator:
 
 If `input` isn't a contiguous iterator, the behavior is undefined.
 
+==== Groups partitioning
+
+The following properties  is introduced to be used
+as a hint that implementation can use get_max_local_range():
```

Comment:
Users might not understand why this is helpful for implementations, so I think we should be a little more explicit here.  Maybe something like:

```suggestion
The following property can be used as a hint that
`get_local_range()` is equivalent to `get_max_local_range()`,
which may enable more aggressive optimizations for some
implementations.

[NOTE]
====
Using `full_group` is necessary to generate SPIR-V block read
and block write instructions, because these instructions are
defined to use the maximum group size as the stride.
====
```